modules/cloud-library/assets/js/editor/component.js

```diff
@@ -64,6 +67,24 @@ export default class Component extends $e.modules.ComponentBase {
 
 		return this.promise;
 	}
+
+	maybeSendQuotaCapacityEvent = ( data ) => {
+		const value = data ? Math.round( ( data.currentUsage / data.threshold ) * 100 ) : 0;
+
+		let quotaUsageAlert = null;
+
+		if ( value < 80 ) {
+			return;
+		} else if ( 80 <= value < 100 ) {
+			quotaUsageAlert = '80%';
+		} else {
+			quotaUsageAlert = '100%';
+		}
+
+		return elementor.templates.eventManager.sendQuotaBarCapacityEvent( {
```

Comment:
```suggestion
	elementor.templates.eventManager.sendQuotaBarCapacityEvent( {
```
Do you need a `return`?